gui/src/installer/step/node/mod.rs

```diff
@@ -0,0 +1,247 @@
+pub mod bitcoind;
+pub mod electrum;
+
+use crate::{
+    hw::HardwareWallets,
+    installer::{
+        context::Context,
+        message::{self, Message},
+        step::{
+            node::{bitcoind::DefineBitcoind, electrum::DefineElectrum},
+            Step,
+        },
+        view, Error,
+    },
+    node::NodeType,
+};
+
+use iced::Command;
+use liana_ui::widget::Element;
+
+#[derive(Clone)]
+pub enum NodeDefinition {
+    Bitcoind(DefineBitcoind),
+    Electrum(DefineElectrum),
+}
+
+impl NodeDefinition {
+    fn new(node_type: &NodeType) -> Self {
```

Comment:
`NodeType` implement `Copy`
```suggestion
    fn new(node_type: NodeType) -> Self {
```